modules/manager/run.go

```diff
@@ -249,8 +249,7 @@ func Run(
 		return fmt.Errorf("unable to set up health check: %w", err)
 	}
 
-	// Enable anonymous reporting when configured but not for development builds
-	// to reduce the noise.
```

Comment:
Why do you consider the comment here to be "outdated"?

Reply:
The check used to be like this:
https://github.com/Kong/gateway-operator/blob/f8d4f259bbc881ce3981dd9a36105ce480268891/modules/manager/run.go#L242-L244